src/NHibernate/Dialect/AnsiSqlKeywords.cs

```diff
@@ -0,0 +1,267 @@
+﻿using System.Collections.Generic;
+
+namespace NHibernate.Dialect
+{
+	/// <summary>
+	/// Maintains the set of ANSI SQL keywords
+	/// </summary>
+	public class AnsiSqlKeywords
+	{
+		/// <summary>
+		/// Retrieve all keywords defined by ANSI SQL:2003
+		/// </summary>
+		public static readonly IReadOnlyCollection<string> Sql2003;
+
+		static AnsiSqlKeywords()
+		{
+			var keywordsSql2003 = new HashSet<string>
```

Comment:
@hazzik, It's not necessary.  The AnsiSqlKeywords are iterated over as an `IEnumerable` when initializing the keyword `HashSet`.